client/src/contexts/Okteto.context.tsx

```diff
@@ -125,6 +145,8 @@ const OktetoProvider = ({ children } : OktetoProviderProps) => {
       output,
       loading,
       ready,
+      status,
+      previousStatus,
```

Comment:
We shouldn't pass the previous status, this should be done by the Environment view who holds the logic to show the toast. To do so, keep a ref to the previous value (as an example of this check: https://blog.logrocket.com/accessing-previous-props-state-react-hooks/)